templates/tfengine/components/iam_members/main.tf

```diff
@@ -55,3 +55,15 @@ module "organization_iam_members" {
   organizations = each.value.resource_ids
   bindings = each.value.bindings
 }
+
```

Comment:
please do a `terraform fmt -recursive` on this file so it's better formatted.

Reply:
done